playbooks/lib_jobs.yml

```diff
@@ -12,7 +12,7 @@
   roles:
     - role: roles/lib_jobs
     - role: roles/datadog
-      when: runtime_env | default('staging') == "production"
+      when: runtime_env | default('staging') == "prod"
```

Comment:
can we please retain production? change the `hosts` file to production and make sure the group vars also do?